matlab_simulation/code v1.0/ME597-6-FastSLAM/newCode/fastSLAM_func.m

```diff
@@ -0,0 +1,174 @@
+function []=fastSLAM_func(PoseInitial,MotionNoise,MeasurementNoise,MapSize,FinalTime,dt,ParticleSize)
```

Comment:
I don't see too many substantial differences between this file and "fastSLAM_func_p.m". I'm guessing one may simply be an older version of the other? Anyway, it would be best to only keep one of these files. If they are different on purpose then there may be a way to combine them and add an additional argument to the function to specify some difference in functionality that can be coded in.